### ISSUE TITLE
[Feature request] Dynamic Line Chart: set Min and Max y axis value from entity Attribute

### ISSUE THREAD

**Issue by FrancescoMenga** [thingsboard/thingsboard#6277]
**Is your feature request related to a problem? Please describe.**
When It comes to configure a Line chart for our projects, we always want the data to be as clear as possible, by setting the correct graph size related to device max and min values. The problem is that we need to always show the "alarm line" (drawn using treshold) of minimum and maximum value. Unfortunately it's not possible to dynamically set min and max value of the chart.

**Describe the solution you'd like**
I would like to set Line Chart minimum and maximum y axis value dinamically.

**Describe alternatives you've considered**
An alternative would be a function, in advanced options, that is used to check device info and calculate min and max y axis value.

**Additional context**

**Comment by gcorthey** [comment]
Was this solved?

**Comment by FrancescoMenga** [comment]
@gcorthey no, even in newer versions of tb line chart min and max value is fixed.
I switched to Echarts to fix the issue but it would be useful to integrate it into the system line charts.

**Comment by mykhailokornieiev** [comment]
Hi. Thank you for the suggestion. Such a feature request was previously added to our backlog for developers. Please follow our [Release Notes](https://thingsboard.io/docs/reference/releases/) to see all new additions and fixes to the platform in subsequent releases.

**Comment by mykhailokornieiev** [comment]
Hi. I'm glad to inform you that this feature was implemented with v4.3 release

<img width="870" height="893" alt="Image" src="https://github.com/user-attachments/assets/e950cb09-10ee-41c7-94fc-dc4eabc029d4" />
